static/scss/course-cards.scss

```diff
@@ -46,6 +46,25 @@
           letter-spacing: 0;
           line-height: 19px;
         }
+
+        .course-card-title {
+          color: $primary;
+
+          &:hover {
+            color: #63131f;
+            text-decoration: underline;
+            cursor: pointer;
+          }
+
+        }
+      }
+
+      .course-link {
```

Comment:
`course-link` is a little ambiguous. Changing this to `course-card-link` would make it clearer that this style applies to the link that wraps the whole card